src/Stratis.Bitcoin/Configuration/Settings/ConnectionManagerSettings.cs

```diff
@@ -132,7 +128,7 @@ public ConnectionManagerSettings(NodeSettings nodeSettings)
                 this.ExternalEndpoint = new IPEndPoint(IPAddress.Loopback, this.Port);
             }
 
-            this.BanTimeSeconds = config.GetOrDefault<int>("bantime", nodeSettings.Network.IsTest() ? DefaultMisbehavingBantimeSecondsTestnet : DefaultMisbehavingBantimeSeconds, this.logger);
+            this.BanTimeSeconds = config.GetOrDefault<int>("bantime", (int)this.CalculateBanTime(nodeSettings.Network), this.logger);
```

Comment:
I think this is a great idea

Reply:
Can we be consistent and do something similar for `MaxTipAge`? There is nothing that I can see to justify different approaches between `MaxTipAge` and `BanTimeSeconds`

Reply:
@quantumagi yeah of course but in another PR. Perhaps you can do this for us?